src/python_routing_v02/fast_reach/mc_reach.pyx

```diff
@@ -783,7 +783,7 @@ cpdef object compute_network_structured_obj(
               buf_view[i][8] = segment.cs
               buf_view[i][9] = segment.s0
               buf_view[i][10] = flowveldepth[segment.id, timestep-1, 0]
-              buf_view[i][11] = flowveldepth[segment.id, timestep-1, 1]
+              buf_view[i][11] = 0.0 #flowveldepth[segment.id, timestep-1, 1]
```

Comment:
Try using `buf_view[i, j]` for all of these memoryview accesses.

Reply:
Fixed...